cloudapi/api_test.go

```diff
@@ -24,18 +24,23 @@ func fprintf(t *testing.T, w io.Writer, format string, a ...interface{}) int {
 func TestCreateTestRun(t *testing.T) {
 	t.Parallel()
 	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		exp := `{"name":"test","vus":0,"thresholds":null,"duration":0}`
+		b, err := io.ReadAll(r.Body)
+		if err != nil || string(b) != exp {
+			http.Error(w, "unexpected request body", http.StatusInternalServerError)
+			return
+		}
```

Comment:
hmm I think the order of marshaling fields in JSON it's not guaranteed, so something like [this](https://pkg.go.dev/github.com/stretchr/testify/assert#JSONEq) is probably better here:
```suggestion
		require.NoError(t, err)
		assert.JSONEq(t, exp, string(b))
```